packages/api/test/__snapshots__/blob.test.ts.snap

```diff
@@ -36,11 +36,11 @@ exports[`Blob router > getAll > when getting expanded blob results > should retu
       "blobGasPrice": "22",
       "blobGasUsed": "5500000",
       "excessBlobGas": "15000",
-      "hash": "blockHash007",
       "slot": 107,
-      "timestamp": "2023-08-31T14:00:00.000Z",
     },
+    "blockHash": "blockHash007",
     "blockNumber": 1007,
+    "blockTimestamp": "2023-08-31T14:00:00.000Z",
```

Comment:
Is this a breaking change for the API users? In this case, we should warn it

Reply:
It shouldn't be a breaking change.

We're keeping the current structure and just adding a few extra fields